src/js/components/homepageUpdate/WordOfTheDay/WordOfTheDay.jsx

```diff
@@ -172,7 +172,7 @@ const WordOfTheDay = () => {
                 selectWordOfTheDay();
                 setGlossary(res.data.results);
                 setLoading(false);
-                setError(false);
+                setError(true);
```

Comment:
Looks like this was missed.  The word of the date error message is always showing now.

Reply:
on it